config/__init__.py

```diff
@@ -20,6 +20,7 @@
 
     BASEDIR = path.abspath(path.join(path.dirname(__file__), pardir))
     SQLALCHEMY_DATABASE_URI = "sqlite:///" + path.join(BASEDIR, "events.db")
+    CELERY_BROKER_URL = 'sqlalchemy+' + SQLALCHEMY_DATABASE_URI
```

Comment:
It's somewhat specified by the tutorial by Celery, I just followed their example on this.